docs/advanced/runtime/docs/other.md

```diff
@@ -33,6 +33,70 @@ Displaying JIT debug output
 
 Use the -v -v -v -v command line argument. Set the MONO_VERBOSE_METHOD env variable to display output for only one method.
 
+
+Dumping JIT IR to IGV
+---------------------
+Set `MONO_CFG_DUMP_METHOD` to specify a method to dump over network to a running instance of the [IdealGrpahVisualizer (IGV)](http://ssw.jku.at/General/Staff/TW/igv.html).
+An IGV build that is compatible with the implementation in Mono is available for [Mac](https://www.dropbox.com/s/j08qtuk7ls2q7j8/idealgraphvisualizer_mac_b18b92bc.zip?dl=0) and [Linux/Windows](https://www.dropbox.com/s/3fwsj81df81qrya/idealgraphvisualizer_others_b18b92bc.zip?dl=0) and requires at least JRE 1.7 to run.
```

Comment:
Yeah, that's a no-go. Do we have a fork of the code on github somewhere? You can add a release with the zip's there.